application_sandbox/pipeline_executable_properties/main.cpp

```diff
@@ -42,8 +42,9 @@ int main_entry(const entry::EntryData* data) {
   vulkan::VulkanApplication app(
       data->allocator(), data->logger(), data, {},
       {VK_KHR_PIPELINE_EXECUTABLE_PROPERTIES_EXTENSION_NAME}, {0}, 1024 * 128,
-      1024 * 128, 1024 * 128, 1024 * 128, false, false, false, 0, false,
-      &pipeline_executable_info_features);
+      1024 * 128, 1024 * 128, 1024 * 128, false, false, false, 0, false, false,
+        VK_COLOR_SPACE_SRGB_NONLINEAR_KHR, false, false, nullptr, false, false,
```

Comment:
Did you mean to include this here? 
It might be worth including the parameter names in comments to make it clear what's going on here, e.g.
...
/\*host_buffer_size=\*/ 1024 * 128,
/\*device_image_size=\*/ 1024 * 128,
/\*device_buffer_size=\*/ 1024 * 128,
/\*coherent_buffer_size=\*/ 1024 * 128,
/\*use_async_compute_queue=\*/ false, 
/\*use_sparse_binding=\*/ false,
...

Reply:
This was included on purpose, but should probably be a separate MR.

However, instead of adding the parameter names, I will take it on myself to clean up this interface completely. This pile of flags has become completely unwieldy, and it needs a better interface.

Reply:
Opened https://github.com/google/vulkan_test_applications/issues/149